src/pubsub.rs

```diff
@@ -101,6 +104,17 @@ impl Future for SubscriptionActive {
         let mut inner = Pin::into_inner(self);
         match &mut inner {
             SubscriptionActive::Ready(res) => Poll::Ready(*res),
+            SubscriptionActive::RequestWithOwner(ref mut req) => match Pin::new(req).poll(cx) {
+                Poll::Pending => Poll::Pending,
+                Poll::Ready((Ok(owner_sub_exists), Ok(sub_exists))) => {
```

Comment:
Wouldn't one Ok(_) be enough? It seems like it wouldn't matter if the other is Err(_)

Reply:
Done